Rocket.Chat/API/Requests/Room/RoomCreateRequest.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  RoomCreateRequest.swift
+//  Rocket.Chat
+//
+//  Created by Bruno Macabeus Aquino on 28/09/17.
+//  Copyright © 2017 Rocket.Chat. All rights reserved.
+//
+//  DOCS:
+//      https://rocket.chat/docs/developer-guides/rest-api/channels/create
+//      https://rocket.chat/docs/developer-guides/rest-api/groups/create
+
+import Foundation
+
+enum RoomCreateType {
+    case channel
+    case group
+
+    var path: String {
+        switch self {
+        case .channel:
+            return "/api/v1/channels.create"
+        case .group:
+            return "/api/v1/groups.create"
+        }
+    }
+}
+
+class RoomCreateRequest: APIRequest {
```

Comment:
On the API, can you use SubscriptionCreate... instead of RoomCreate... and put the files under Subscription? That's the naming we're using for the API and we don't want to introduce conflicting nomenclature.